modules/claims_api/app/workers/claims_api/ews_updater.rb

```diff
@@ -16,11 +16,16 @@ def perform(ews_id)
       ews = ClaimsApi::EvidenceWaiverSubmission.find(ews_id)
       bgs_claim = benefit_claim_service(ews).find_bnft_claim(claim_id: ews.claim_id)
 
-      unless bgs_claim&.dig(:bnft_claim_dto, :filed5103_waiver_ind) == FILE_5103
+      if bgs_claim&.dig(:bnft_claim_dto, :filed5103_waiver_ind).present? &&
+         bgs_claim&.dig(:bnft_claim_dto, :filed5103_waiver_ind) != FILE_5103
         bgs_claim[:bnft_claim_dto][:filed5103_waiver_ind] = FILE_5103
-
         update_bgs_claim(ews, bgs_claim)
       end
+      if bgs_claim&.dig(:bnft_claim_dto, :filed5103_waiver_ind).blank?
```

Comment:
we could set `bgs_claim&..dig(:bnft_claim_dto, :filed5103_waiver_ind)` to a variable and check that so we're not repeating this and forcing it to multi-lines (well, maybe. might still be long enough dependent on the variable name)? more thought that a needed adjustment. 21 would remain the same though.